### PR TITLE
Master test

### PR DESCRIPTION
**Pull request by Raghu-Srungavarapu**
1) Import error : Corrected the import statement in user_interfaces\__init__.py. "fromelection_anomaly.db_routinesimportCreate_CDF_dbasdb_cdf"
	2) 020_load_jurisdiction_to_db.py: Doesn't create a new DB when it runs in noninteractive mode. 
	3) Added src/Temp/ to git ignore

### CODE REVIEW

File: src/election_anomaly/user_interface/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from election_anomaly import db_routines
 from election_anomaly import db_routines as dbr
-from election_anomaly.db_routines import create_cdf_db as db_cdf
+from election_anomaly.db_routines import Create_CDF_db as db_cdf
```
Comment (ericmtsai) [inline src/election_anomaly/user_interface/__init__.py:5]: I don't believe this is correct. The latest version of master should only ever reference `create_cdf_db`, not `Create_CDF_db`. If you are using the `python setup.py install` approach, you may need to rebuild the package and try again (in fact, you'll need to rebuild each time you make _any_ change).

**Comment by Raghu-Srungavarapu** [comment]
Reverted import statement changes.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 *.pyc
 src/tmp/
+src/Temp/
```
Comment (ericmtsai) [inline .gitignore:3]: @Raghu-Srungavarapu What is this folder needed for?

Comment (Raghu-Srungavarapu) [reply .gitignore:3]: Actually, when I think of it again, This doesn't seem like required line,
Previously I used this to stores the output files from top counts.  However, the path to this folder is user-defined. I am going to change my path to local and delete this row.

**Pull request closed without merging**